requirements.txt

```diff
@@ -1,4 +1,5 @@
-viktor==14.0.0
+viktor==14.13.0
 folium==0.14.0
 geopandas==0.12.2
 OWSLib==0.28.1
```

Comment:
There are newer versions of:
- OWSLib: https://pypi.org/project/OWSLib/
- geopandas: https://pypi.org/project/geopandas/
- folium: https://pypi.org/project/folium/

If you update to the latest this solves deprecation warnings